bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/NonSiderealTargetRulesSpec.scala

```diff
@@ -117,7 +117,7 @@ class NonSiderealTargetRulesSpec extends RuleSpec {
   def obs(target: Target, sb: Option[SchedulingBlock], inst: SPComponentType = InstGmosSouth.SP_TYPE): ISPObservation = {
     val f = POTUtil.createFactory(UUID.randomUUID())
     val p = f.createProgram(null, SPProgramID.toProgramID("no-site"))
-    val o = f.createObservation(p, null) <| { on =>
+    val o = f.createObservation(p, Instrument.fromComponentType(inst), null) <| { on =>
       p.addObservation(on)
       on.getDataObject.asInstanceOf[SPObservation] <| { o =>
```

Comment:
Just as a random aside of curiosity, I always see that `IdOpts` `<|` will be becoming obsolete soon. Should this be something we should concern ourselves with?

Reply:
With some luck we won't ever have to upgrade scalaz. But if we do we can add this syntax back ourselves.

Reply:
I doubt we'll ever upgrade scalaz in ocs